### PR TITLE
Replace var with let

### PR DESCRIPTION
**Pull request by adrianheine**
This also adds bublé to the rollup call. Build time with watch roughly doubles, so it's usually between 300 and 600 for me.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import buble from 'rollup-plugin-buble';
+
+export default {
+  banner: `\/\/ CodeMirror, copyright (c) by Marijn Haverbeke and others
```
Comment (marijnh) [inline rollup.config.js]: Why the escaped slashes?

Comment (adrianheine) [inline rollup.config.js]: I got myself intimidated by vim's apparently wrong syntax highlighting.

Comment (marijnh) [inline rollup.config.js]: Ah, right. The situation is no better in Emacs' js-mode, unfortunately.

**Comment by marijnh** [comment]
So when I merge this the resulting codemirror.js fails on parsing in Chrome with

```
codemirror.js:4674 Uncaught SyntaxError: for-in loop variable declaration may not have an initializer.
```

Which appears to be a bug in Bublé that I myself introduced. I'll look into it.

**Comment by marijnh** [comment]
There's already a Bublé patch at https://gitlab.com/Rich-Harris/buble/merge_requests/85/diffs . Until that gets merged and released, I've changed that single `let` back to `var` in 6992c53, and merged this.

**Pull request closed without merging**